### DISCUSSION TITLE
请问frp支持ws协议吗

### DISCUSSION THREAD

**Discussion by A-Kevin1217** [natfrp/wiki#463] [使用问题]
因为很需要

[a comment by an automated account is not shown]

**Reply by A-Kevin1217** [reply]
![image](https://github.com/natfrp/wiki/assets/21151999/498133aa-f83d-4a75-8ec4-e0d321b9ae55)
这个是为啥捏

**Reply by fakeboboliu** [reply]
> 这个是为啥捏

我们从您的截图中得不到足够有效信息，请考虑检查：
 - 启动器日志
 - 对于您的场景来说，F12
 - 对于浏览器来说，考虑mixed-content
 - 检查应用协议是否正确

**Comment by tencentbook** [comment]
被封锁了一些协议导致的，可以向所在地区的人大代表提出意见取消网络限制

**Reply by A-Kevin1217** [reply]
还有其他解决方式吗

**Reply by tencentbook** [reply]
才疏学浅，而且就算能做到也不能做吧，国家限制了你还要突破限制有风险

**Reply by A-Kevin1217** [reply]
> 才疏学浅，而且就算能做到也不能做吧，国家限制了你还要突破限制有风险

好的谢谢
